app/js/components/common/Header.js

```diff
@@ -5,119 +5,119 @@ import {ApiHelper} from '../../helpers/apiHelper';
 const NUMBER_OF_COLUMNS = 3;
```

Comment:
@andela-jomadoye Great work with the indents. But does this comply to the 2 indent space rule you stated? same to line 7 and line % on SavedCohorts.js. According to what I can see, some have one space indenting.